Packages/BaseAtoms/Runtime/Events/VoidEvent.cs

```diff
@@ -7,8 +7,7 @@ namespace UnityAtoms.BaseAtoms
     /// </summary>
     [EditorIcon("atom-icon-cherry")]
     [CreateAssetMenu(menuName = "Unity Atoms/Events/Void", fileName = "VoidEvent")]
-    public sealed class VoidEvent : AtomEvent<Void>
-    {
+    public sealed class VoidEvent : AtomEvent<Void>     {
```

Comment:
is not valid based on our https://github.com/AdamRamberg/unity-atoms/blob/master/.editorconfig

make sure your IDE uses the editorconfig (vs code needs a plugin for that IIRC)

Reply:
invalid formatting makes up most of the changeset. please correct.

Reply:
Yeah - I need to fix this in the generator mostly - not much point in having it generate code that does not need to be modified if it then relies on the IDE to fix the formatting anyway.  I was having trouble getting linefeeds and whitespace to behave where blocks were removed.

Failing that, I could remove the generated files from the PR - I'll see if I can avoid that though.  I'll take a run at it this weekend.